test/unit/org/apache/cassandra/index/sai/cql/VectorTypeTest.java

```diff
@@ -61,10 +61,6 @@ private void verifyChecksum() {
         ColumnFamilyStore cfs = Keyspace.open(KEYSPACE).getColumnFamilyStore(currentTable());
         cfs.indexManager.listIndexes().stream().forEach(index -> {
             var indexContext = SAITester.createIndexContext(index.getIndexMetadata().name, VectorType.getInstance(FloatType.instance, 100), cfs);
```

Comment:
can get actual existing indexContext via
```java
var indexContext = (IndexContext)FieldUtils
                                   .getDeclaredField(index.getClass(), "indexContext", true)
                                   .get(index);
```

Reply:
Is it better to get the real context? It seems like it to me.

Reply:
Done